src/main/java/zmaster587/advancedRocketry/client/render/planet/RenderPlanetarySky.java

```diff
@@ -796,9 +797,11 @@ public static void renderPlanetPubHelper(Tessellator tessellator1, ResourceLocat
 
 	private void drawStar(Tessellator buffer, int solarOrbitalDistance, float sunSize, float r, float g, float b, float multiplier) {
 		//Set sun color and distance
+		IPlanetaryProvider planetaryProvider = (IPlanetaryProvider)mc.theWorld.provider;
```

Comment:
We probably don't want these two calls here, "planetaryProvider" and "properties" are not used.

Reply:
I think I added those for some reason that then was removed. Sorry!